homeassistant/components/shelly/sensor.py

```diff
@@ -396,6 +392,37 @@ def __init__(
         entity_category=EntityCategory.DIAGNOSTIC,
         removal_condition=lambda _, block: block.valve == "not_connected",
     ),
+    ("sensor", "gas"): BlockSensorDescription(
+        key="sensor|gas",
+        name="Gas detected",
+        translation_key="gas_detected",
+        device_class=SensorDeviceClass.ENUM,
+        options=[
+            "none",
+            "mild",
+            "heavy",
+            "test",
+            "unknown",
```

Comment:
I'm not sure if we should treat `unknown` as the standard sensor state since `unknown` state in HA means `no state`. Maybe the sensor should return `None` as a state when the device reports `unknown`?

Reply:
This is a new entity and should be implemented correctly from the start.

Reply:
which in the end was not true for the other two ;-)

Reply:
https://github.com/home-assistant/core/pull/140597